forwarder/kafka/kafka_helpers.py

```diff
@@ -11,16 +11,50 @@
 
 from .kafka_producer import KafkaProducer
 
+DEFAULT_SASL_MECHANISM = "SCRAM-SHA-256"
+
+
+def get_sasl_config(
+    mechanism: str, username: Optional[str] = None, password: Optional[str] = None
```

Comment:
maybe not optional

Reply:
We decide to leave it like this for now. Consider also comment https://github.com/ess-dmsc/forwarder/pull/173#discussion_r983304634
(if we refactor the way we parse configuration parameters the confusion may go away)